galaxy_api/api/ui/viewsets/collection.py

```diff
@@ -189,7 +189,11 @@ class CollectionImportViewSet(viewsets.GenericViewSet):
     filterset_class = CollectionImportFilter
 
     def list(self, request, *args, **kwargs):
-        queryset = self.filter_queryset(self.get_queryset())
+        sort = self.request.query_params.get('sort', None)
```

Comment:
This works, and I personally think this is a nice clean way to implement this, but a more DRF like approach would probably be to override the `self.get_queryset()` function, kind of like we do here: https://github.com/ansible/galaxy/blob/028a0ae075d5497ef43b6704c431a6f4c19b4162/galaxy/api/views/email.py#L50

Lets see what @cutwater has to say.

Reply:
This implementation just passes sort parameter to query.
 It does not check if this parameter is valid or not, if this parameter value is allowed or not.

Django REST Framework provides [OrderingFilter](https://www.django-rest-framework.org/api-guide/filtering/#orderingfilter) which allows specifying columns for ordering explicitly.
Query parameter name can be customised with `api_settings.ORDERING_PARAM`.

I encourage you to always use an explicit list of allowed fields for ordering or in similar case.